about-gordon.html

```diff
@@ -0,0 +1,99 @@
+
+    <p>In the UK, the Gordon Ramsay Group has opened new locations across London, including <span class="important">Union Street Café</span>, <span class="important">London House</span>, <span class="important">Heddon Street Kitchen</span> and two further <span class="important">maze Grills</span>, one of which is located on the former Aubergine site on Park Walk, Chelsea.</p>
+
+    <h3>Gordon Ramsay on TV</h3>
+
+    <p>Ramsay has also become a star of the small screen both in the US and internationally as a TV chef, with four top-rated FOX shows that air in more than 200 territories worldwide. These critically-acclaimed shows include <span class="important">Hell's Kitchen</span>, <span class="important">MasterChef</span>, <span class="important">MasterChef Junior</span>, and <span class="important">Hotel Hell</span>, as well as UK shows: <span class="important">Gordon Behind Bars</span>, <span class="important">Ultimate Cookery Course</span>, <span class="important">Gordon's Great Escapes</span> and <span class="important">Gordon Ramsay: Shark Baits</span>.</p>
+
+    <h3>Other Ventures</h3>
+
+    <p>While best known as a celebrity chef now, Ramsay has seen success in a variety of other ventures outside of food and television too. Most notably as a published author, Ramsay has released a number of books, many of which have become best-sellers around the world; including his autobiography, <span class="important">Roasting in Hell's Kitchen</span>. Aside from this, Ramsay has a <span class="important">global partnership with WWRD (Waterford, Wedgwood, Royal Doulton)</span>, which offers quality home and lifestyle products.</p>
+
+    <p>In 2006, in recognition of all that Ramsay has contributed to the industry, he received an <span class="important">OBE (Order of the British Empire)</span>, awarded by Queen Elizabeth II. Since then, Ramsay and his wife, Tana, set up the <span class="important">Gordon and Tana Ramsay Foundation</span> in 2014, to make a meaningful difference to the charities that are important to them. You can find out more about the Gordon and Tana Ramsay Foundation <a href="http://www.gosh.org/get-involved/philanthropy/how-you-can-support-us/gordon-and-tana-ramsay-foundation">here</a>.</p>
+
+    <footer>
+
+    <div class="icon-container">
```

Comment:
Consider using a general purpose container all the rows to constrain the content to some reasonably max-width as well as a container for all the icons here that will group them all as 1 unit making them easier to manipulate such items if you choose to make more complex layouts.